### DISCUSSION TITLE
where to get "signature" in tokenType.ts of each token?

### DISCUSSION THREAD

**Discussion by Sotatek-ToanTran** [CoolBitX-Technology/coolwallet-signing-tools#33] [Q&A]
- Description

we are deploying to tokenTypes, in the file tokenType.ts there is a list of tokens. Except for the "signature" property, we don't know how it is generated for each token.

- Screenshots or appendix

![CleanShot 2022-01-18 at 12 19 16](https://user-images.githubusercontent.com/31679583/149875487-3f853710-bd65-4d81-91cb-9a85f1bf84b9.png)

**Comment by srt1678** [comment]
The token signature is generated by the coolwallet team. We use this signature to validate whether the given script can be executed. 
Signature was generated by CoolBitx. We use this signature to validate whether the given script can be executed. For development card the signature is `"FA".padEnd(144, '0')`.

**Reply by Sotatek-KhanhLe** [reply]
Was that signature you mention is script signature ? Because as we saw token type signature is using for argument not script
![image](https://user-images.githubusercontent.com/88299386/150052192-40d466bb-ecd1-410e-9579-fab5c555c5e0.png)
